### ISSUE TITLE
A11y_debug-adapter-protocol_AI4W: Ensure links are distinguished from surrounding text in a way that does not rely on color (a)

### ISSUE THREAD

**Issue by Yashi1201** [microsoft/debug-adapter-protocol#375]
[Check out Accessibility Insights!](url) - Identify accessibility bugs before check-in and make bug fixing faster and easier."

**Github Tags:** #A11yMAS;#A11ySev2;#A11yTCS;#AI4W;#Benchmark;#ChromiumEdge;#DesktopWeb;#FTP;#WCAG 1.4.1;#UseOfColor;#Win10;#link-in-text-block;#P2_debugadapterprotocol_Web_Sep2021;#Visual Studio Code Client;#Regressed:03-29-23;#Reopened;

## Environment Details:
Application Name: Debug Adapter Protocol Page
#URL: https://microsoft.github.io/debug-adapter-protocol/specification
Windows Version: Win10 22H2 19045.2251
Chromium Edge version: Version 110.0.1587.49 (Official Build) (64-bit)
## Repro Steps:

1. **Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility tester after verification of the fix**
2. Launch the url: https://microsoft.github.io/debug-adapter-protocol/specification
3. Run accessibility insights for web and see the results.
## Issue: 
Ensure links are distinguished from surrounding text in a way that does not rely on color (link-in-text-block - https://dequeuniversity.com/rules/axe/4.6/link-in-text-block?application=msftAI)

Target application: Specification - https://microsoft.github.io/debug-adapter-protocol/specification

Element path: p:nth-child(2) > a

Snippet: <a href="/debug-adapter-protocol/debugAdapterProtocol.json">here</a>
### Similar issue is observed in scenario:-

1. https://microsoft.github.io/debug-adapter-protocol/specification.html#Events_Output
2. https://microsoft.github.io/debug-adapter-protocol/implementors/adapters/
3. https://microsoft.github.io/debug-adapter-protocol/overview

## How to fix: 
Fix any of the following:
  The link has insufficient color contrast of 1.87:1 with the surrounding text. (Minimum contrast is 3:1, link text: #2753e3, surrounding text: #373a3c)
  The link has no styling (such as underline) to distinguish it from the surrounding text

Environment: Microsoft Edge version 110.0.1587.49

====

This accessibility issue was found using Accessibility Insights for Web 2.37.3 (axe-core 4.6.3), a tool that helps find and fix accessibility issues. Get more information & download this tool at http://aka.ms/AccessibilityInsights.
## User Impact:
If there is no visual under lining to the links even without hovering mouse, Monochromatic users will not be able to distinguish between normal text and a link so they could miss the mouse hovering to get the under lining, over the actual links considering them as a plain text.
## Attachement:
![375_A11y_debug-adapter-protocol](https://user-images.githubusercontent.com/93374043/220114652-8ce470d8-f0cd-4ce9-8e81-a97b7a6d2401.png)
![375_A11y_debug-adapter-protocol_without ai4w](https://user-images.githubusercontent.com/93374043/220114695-3a96a501-26fb-4e5e-8eb6-9f1be59ed491.png)

**Comment by rzhao271** [comment]
Fixed by https://github.com/microsoft/debug-adapter-protocol/pull/380

**Comment by Yashi1201** [comment]
Verified the issue in env https://microsoft.github.io/debug-adapter-protocol/specification
Main issue is still repro
![image](https://user-images.githubusercontent.com/93374043/228438232-5a7b1cbe-b2b8-41da-981e-55dcd5609925.png)

Similar issues are also repro
![image](https://user-images.githubusercontent.com/93374043/228438552-c9f3323b-446d-4f73-8922-fa40fdf3ef31.png)

![image](https://user-images.githubusercontent.com/93374043/228438882-6171e05b-634f-441b-90e3-0f52d682fd82.png)

![image](https://user-images.githubusercontent.com/93374043/228438997-236e8ff4-39ed-4703-b933-1dfbcb837089.png)

Please reopen the bug as the issue is still not fixed.

**Comment by rzhao271** [comment]
Should be good now. I pushed in another patch today.

**Comment by fsteffi** [comment]
#Closed;
no issue
https://microsoft.github.io/debug-adapter-protocol/specification
![image](https://user-images.githubusercontent.com/24580570/228808427-b9a28351-2950-423c-91ee-5de1e2b52242.png)
